fta/samples/utils.py

```diff
@@ -15,8 +16,31 @@ def humansize(nbytes):
     return "%s %s" % (f, suffixes[i])
 
 
+def convert_fathom_sample_to_labeled_sample(fathom_sample, suffix=".html"):
+    soup = BeautifulSoup(fathom_sample, features="html.parser")
+
+    # remove any existing data-fta_id attrs first
+    fta_labeled_elements = [
+        item for item in soup.find_all() if "data-fta_id" in item.attrs
+    ]
+    for element in fta_labeled_elements:
+        del element["data-fta_id"]
+
+    fathom_labeled_elements = [
+        item for item in soup.find_all() if "data-fathom" in item.attrs
+    ]
+    label_to_fta_id = {}
```

Comment:
A page can have multiple elements tagged with the same label:

```python
In [7]: convert_fathom_sample_to_labeled_sample('<input data-fta_id="aaa" data-fathom="email"/><input data-fta_id="bbb" data-fathom="email"/>')
Out[7]:
('<input data-fta_id="9fc8e7a6-40fe-4a94-bfb7-76ab96e46b94"/><input data-fta_id="3133a4c9-855e-4613-b162-bc517e21145f"/>',
 {'email': UUID('3133a4c9-855e-4613-b162-bc517e21145f')})
```

So this structure should either be fta_id_to_label or a list.

Reply:
Yep, missed that scenario.  Thanks.

Reply:
Done